docs(*) auto-generated docs update for 2.1.x with manual changes (#2130)
Kong/docs.konghq.com: app/2.1.x/db-less-admin-api.md

    `connect_timeout`<br>*optional* |  The timeout in milliseconds for establishing a connection to the upstream server.  Defaults to `60000`.
    `write_timeout`<br>*optional* |  The timeout in milliseconds between two successive write operations for transmitting a request to the upstream server.  Defaults to `60000`.
    `read_timeout`<br>*optional* |  The timeout in milliseconds between two successive read operations for transmitting a request to the upstream server.  Defaults to `60000`.
    `tags`<br>*optional* |  An optional set of strings associated with the Service, for grouping and filtering. 
    `client_certificate`<br>*optional* |  Certificate to be used as client certificate while TLS handshaking to the upstream server. With form-encoded, the notation is `client_certificate.id=<client_certificate id>`. With JSON, use "`"client_certificate":{"id":"<client_certificate id>"}`.

>     `tls_verify`<br>*optional* |  Whether to enable verification of upstream server TLS certificate. If set to `null` then Nginx default is respected. 

Reviewer: ```suggestion
    `tls_verify`<br>*optional* |  Whether to enable verification of upstream server TLS certificate. If set to `null`, then the Nginx default is respected. 
```

    `tls_verify_depth`<br>*optional* |  Maximum depth of chain while verifying upstream server's TLS certificate. If set to `null` when Nginx default is respected.  Defaults to `null`.

>     `ca_certificates`<br>*optional* |  Array of `CA Certificate` object UUIDs that are used to build the trust store while verifying upstream server's TLS certificate. If set to `null` when Nginx default is respected. If default CA list in Nginx are not specified and TLS verification is enabled, then handshake with upstream server will always fail (because no CA are trusted).  With form-encoded, the notation is `ca_certificates[]=4e3ad2e4-0bc4-4638-8e34-c84a417ba39b&ca_certificates[]=51e77dc2-8f3e-4afa-9d0e-0e3bbbcfd515`. With JSON, use an Array.

Reviewer: ```suggestion
    `ca_certificates`<br>*optional* |  Array of `CA Certificate` object UUIDs that are used to build the truststore while verifying upstream server's TLS certificate. If set to `null`, then the Nginx default is respected. If default CA list in Nginx are not specified and TLS verification is enabled, then handshake with the Upstream server will always fail (because no CA are trusted).  With form-encoded, the notation is `ca_certificates[]=4e3ad2e4-0bc4-4638-8e34-c84a417ba39b&ca_certificates[]=51e77dc2-8f3e-4afa-9d0e-0e3bbbcfd515`. With JSON, use an array.
```

>     `url`<br>*shorthand-attribute* |  Shorthand attribute to set `protocol`, `host`, `port` and `path` at once. This attribute is write-only (the Admin API never "returns" the url). 

Reviewer: ```suggestion
    `url`<br>*shorthand-attribute* |  Shorthand attribute to set `protocol`, `host`, `port` and `path` at once. This attribute is write-only (the Admin API never returns the URL). 
```

consumer_body: |
    Attributes | Description
    ---:| ---

>     `username`<br>*semi-optional* |  The unique username of the consumer. You must send either this field or `custom_id` with the request. 

Reviewer: ```suggestion
    `username`<br>*semi-optional* |  The unique username of the Consumer. You must send either this field or `custom_id` with the request. 
```

>     `custom_id`<br>*semi-optional* |  Field for storing an existing unique ID for the consumer - useful for mapping Kong with users in your existing database. You must send either this field or `username` with the request. 

Reviewer: ```suggestion
    `custom_id`<br>*semi-optional* |  Field for storing an existing unique ID for the Consumer - useful for mapping Kong with users in your existing database. You must send either this field or `username` with the request. 
```

>     `tags`<br>*optional* |  An optional set of strings associated with the Consumer, for grouping and filtering. 

Reviewer: ```suggestion
    `tags`<br>*optional* |  An optional set of strings associated with the Consumer for grouping and filtering. 
```

plugin_body: |
    Attributes | Description
    ---:| ---

>     `name` |  The name of the Plugin that's going to be added. Currently the Plugin must be installed in every Kong instance separately. 

Reviewer: ```suggestion
    `name` |  The name of the Plugin that's going to be added. Currently, the Plugin must be installed in every Kong instance separately. 
```

    `consumer`<br>*optional* |  If set, the plugin will activate only for requests where the specified has been authenticated. (Note that some plugins can not be restricted to consumers this way.). Leave unset for the plugin to activate regardless of the authenticated consumer.  Defaults to `null`.With form-encoded, the notation is `consumer.id=<consumer id>` or `consumer.username=<consumer username>`. With JSON, use "`"consumer":{"id":"<consumer id>"}` or `"consumer":{"username":"<consumer username>"}`.
    `config`<br>*optional* |  The configuration properties for the Plugin which can be found on the plugins documentation page in the [Kong Hub](https://docs.konghq.com/hub/). 
    `protocols` |  A list of the request protocols that will trigger this plugin. The default value, as well as the possible values allowed on this field, may change depending on the plugin type. For example, plugins that only work in stream mode will only support `"tcp"` and `"tls"`.  Defaults to `["grpc", "grpcs", "http",`<wbr>` "https"]`.
    `enabled`<br>*optional* | Whether the plugin is applied. Defaults to `true`.

>     `tags`<br>*optional* |  An optional set of strings associated with the Plugin, for grouping and filtering. 

Reviewer: ```suggestion
    `tags`<br>*optional* |  An optional set of strings associated with the Plugin for grouping and filtering. 
```

    ---:| ---
    `cert` | PEM-encoded public certificate chain of the SSL key pair.
    `key` | PEM-encoded private key of the SSL key pair.

>     `tags`<br>*optional* |  An optional set of strings associated with the Certificate, for grouping and filtering. 

Reviewer: ```suggestion
    `tags`<br>*optional* |  An optional set of strings associated with the Certificate for grouping and filtering. 
```

>     `snis`<br>*shorthand-attribute* |  An array of zero or more hostnames to associate with this certificate as SNIs. This is a sugar parameter that will, under the hood, create an SNI object and associate it with this certificate for your convenience. To set this attribute this certificate must have a valid private key associated with it. 

Reviewer: ```suggestion
    `snis`<br>*shorthand-attribute* |  An array of zero or more hostnames to associate with this certificate as SNIs. This is a sugar parameter that will, under the hood, create an SNI object and associate it with this certificate for your convenience. To set this attribute, the certificate must have a valid private key associated with it. 
```

    Attributes | Description
    ---:| ---
    `cert` | PEM-encoded public certificate of the CA.

>     `cert_digest`<br>*optional* | SHA256 hex digest of the public certificate

Reviewer: ```suggestion
    `cert_digest`<br>*optional* | SHA256 hex digest of the public certificate.
```

>     `tags`<br>*optional* |  An optional set of strings associated with the Certificate, for grouping and filtering. 

Reviewer: ```suggestion
    `tags`<br>*optional* |  An optional set of strings associated with the Certificate for grouping and filtering. 
```

        "id": "43429efd-b3a5-4048-94cb-5cc4029909bb",
        "created_at": 1422386534,
        "cert": "-----BEGIN CERTIFICATE-----...",
        "cert_digest": "c641e28d77e93544f2fa87b2cf3f3d51...",
        "tags": ["user-level", "low-priority"]
    }, {
        "id": "d26761d5-83a4-4f24-ac6c-cff276f2b79c",
        "created_at": 1422386534,
        "cert": "-----BEGIN CERTIFICATE-----...",
        "cert_digest": "c641e28d77e93544f2fa87b2cf3f3d51...",
        "tags": ["admin", "high-priority", "critical"]
    }],

sni_body: |
    Attributes | Description
    ---:| ---
    `name` | The SNI name to associate with the given certificate.

>     `tags`<br>*optional* |  An optional set of strings associated with the SNIs, for grouping and filtering. 

Reviewer: ```suggestion
    `tags`<br>*optional* |  An optional set of strings associated with the SNIs for grouping and filtering. 
```

    `healthchecks.passive.`<wbr>`healthy.successes`<br>*optional* | Number of successes in proxied traffic (as defined by `healthchecks.passive.healthy.http_statuses`) to consider a target healthy, as observed by passive health checks. Defaults to `0`.
    `healthchecks.passive.`<wbr>`healthy.http_statuses`<br>*optional* | An array of HTTP statuses which represent healthiness when produced by proxied traffic, as observed by passive health checks. Defaults to `[200, 201, 202, 203, 204, 205,`<wbr>` 206, 207, 208, 226, 300, 301,`<wbr>` 302, 303, 304, 305, 306, 307,`<wbr>` 308]`. With form-encoded, the notation is `http_statuses[]=200&http_statuses[]=201`. With JSON, use an Array.
    `healthchecks.threshold`<br>*optional* | The minimum percentage of the upstream's targets' weight that must be available for the whole upstream to be considered healthy. Defaults to `0`.

>     `tags`<br>*optional* |  An optional set of strings associated with the Upstream, for grouping and filtering. 

Reviewer: ```suggestion
    `tags`<br>*optional* |  An optional set of strings associated with the Upstream for grouping and filtering. 
```

target_body: |
    Attributes | Description
    ---:| ---
    `target` |  The target address (ip or hostname) and port. If the hostname resolves to an SRV record, the `port` value will be overridden by the value from the DNS record. 
    `weight`<br>*optional* |  The weight this target gets within the upstream loadbalancer (`0`-`1000`). If the hostname resolves to an SRV record, the `weight` value will be overridden by the value from the DNS record.  Defaults to `100`.

>     `tags`<br>*optional* |  An optional set of strings associated with the Target, for grouping and filtering. 

Reviewer: global comment-don't need a comma here because it creates an incomplete clause.

```suggestion
    `tags`<br>*optional* |  An optional set of strings associated with the Target for grouping and filtering. 
```
Or reorder all of these:

An optional set of strings **for grouping and filtering** that are associated with the Target.
